protocol/interop/interop-launch-sequence.md

```diff
@@ -0,0 +1,55 @@
+# Interop Launch Sequence
+
+## Purpose
+
+This document describes a proposed sequence of events that will occur when interop goes live on
+mainnet. The exact sequence is subject to change, but this document is meant to provide a scaffold
+that helps drive other design documents.
+
+## Proposed Sequence
+
+- Far before the launch of interop, for all chains, in an OPCM action per chain:
+  - An `ETHLockbox` contract is deployed for the chain.
+  - The Upgrade Controller approves the `OptimismPortal` address to use the `ETHLockbox`.
+  - Chains are upgraded to the new `OptimismPortal` implementation.
+    - Because `usingSuperRoots` is `false` by default, this will not result in any immediate impact
```

Comment:
Can you add some links? What is `usingSuperRoots`?

Reply:
Context is here: https://github.com/ethereum-optimism/design-docs/pull/205